scripts/globals/server.lua

```diff
@@ -16,4 +16,23 @@ end
 xi.server.onTimeServerTick = function()
 end
 
+-- Message for use with SmallPacket0x04B
+xi.server.getServerMessage = function(language)
+    local serverMessage = ""
+
+    if language == 0x02 then -- English
```

Comment:
Just a thought for the future, but should we eventually create an enum for language?

Reply:
Good idea, added now